tests/resources/db_variables.robot

```diff
@@ -3,7 +3,7 @@ ${EtcRoot}  /tmp/etc
 ${VarRoot}  /tmp/var
 ${DBName}	centreon_storage
 ${DBNameConf}	centreon
-${DBHost}	localhost
+${DBHost}	127.0.0.1
```

Comment:
I don't understand this change.
localhost => unix socket
127.0.0.1 => tcp socket

Reply:
used to connect to mariadb container